web/src/components/CanvasBoard.tsx

```diff
@@ -0,0 +1,18 @@
+import styles from './CanvasBoard.module.css';
+
+type Props = {
```

Comment:
one file with all types in would be better so refering to types is done in one file rather than multiple types all over the place